src/App.tsx

```diff
@@ -1,46 +1,100 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import "./App.scss";
 import CreateToDo from "./components/CreateToDo";
 import Header from "./components/Header";
 import Todo from "./components/Todo";
 
+interface itemsTypes {
+  todoId: number;
+  content: string;
+  isChecked: boolean; //not working yet
+  isEmpty: boolean;
+}
+
 const App = () => {
-  const [items, setItems] = useState<(string | null)[]>([
-    "Item 1",
-    "Item 2",
-    "Item 3",
+  const [items, setItems] = useState<itemsTypes[]>([
+    {
+      todoId: 1,
+      content: "Item 1",
+      isChecked: false,
+      isEmpty: false,
+    },
+    {
+      todoId: 2,
+      content: "Item 2",
+      isChecked: false,
+      isEmpty: false,
+    },
+    {
+      todoId: 3,
+      content: "Item 3",
+      isChecked: false,
+      isEmpty: false,
+    },
   ]);
 
-  const [hasNull, setHasNull] = useState(false);
+  console.log(items);
+
+  const getHighestId = () => {
+    const itemsId = items.map((item) => item.todoId);
+    return Math.max(...itemsId) + 1;
+  };
+
+  // console.log(getHighestId());
```

Comment:
Could you also remove this unnecessary comment?
```suggestion

```

Reply:
Done.